### PR TITLE
fix: create embedded fs for web assets

### PR DESCRIPTION
**Pull request by eze-kiel**
- fix: create embedded fs for web assets
- fix: fix typo in flag

### CODE REVIEW

File: webui/webui.go [hunk 1]
```diff
@@ -16,6 +17,10 @@ import (
 	"k8s.io/client-go/util/retry"
 )
 
+// assets holds our static web server assets.
+//go:embed assets/*
+var assets embed.FS
```
Comment (xakraz) [inline webui/webui.go:22]: ❓ How does it work?

Comment (eze-kiel) [reply webui/webui.go:22]: the embedded FS is populated at compile-time, which mean that our standalone binary contains all our assets

Comment (eze-kiel) [reply webui/webui.go:22]: (stdlib doc about `embed` package: https://pkg.go.dev/embed)

Comment (xakraz) [reply webui/webui.go:22]: Nice.
Is is something to be aware of while developing locally ?

**Review by xakraz — APPROVED** [review]
LGTM

File: webui/webui.go (hunk 1 above)
Comment (eze-kiel) [reply webui/webui.go:22]: I don't think so, as it occurs during compilation. It's the exact same build command.

> It should have 0 impact on your workflows

:awesome:

**Pull request merged**